packages/dev-tools-pages/ts/components/InlineCode.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { colors } from '../variables';
+
+interface InlineCodeProps {
+    isAlt?: boolean;
+    children: React.ReactNode;
+}
+
+const InlineCode = styled(({ isAlt, children, ...props }: InlineCodeProps) => <code {...props}>{children}</code>)`
+    background-color: ${props => (props.isAlt ? '#E5E8E9' : colors.blueGray)};
```

Comment:
If you define as a separate component and use `React.StatelessComponent<InlineCodeProps>` you don't need to define children.